tests/cluster/tests/29-slot-migration-response.tcl

```diff
@@ -0,0 +1,66 @@
+# Tests for the response of slot migrations.
+
+source "../tests/includes/init-tests.tcl"
+source "../tests/includes/utils.tcl"
+
+# TODO: This test currently tests the ASK and MOVED replies, need to add a test to check for the UNSTABLE reply
+
+test "Create a 10 nodes cluster" {
+    create_cluster 10 0
+    config_set_all_nodes cluster-allow-replica-migration no
+}
+
+test "Cluster is up" {
+    assert_cluster_state ok
+}
+
+set cluster [redis_cluster 127.0.0.1:[get_instance_attrib redis 0 port]]
+catch {unset nodefrom}
+catch {unset nodeto}
+
+$cluster refresh_nodes_map
+
+test "Set many keys" {
+    for {set i 0} {$i < 40000} {incr i} {
+        $cluster set $i $i
+    }
+}
```

Comment:
I think we only need one key or a few keys in the slot we are testing. We can use a hard-coded key and use CLUSTER KEYSLOT to find the slot.

Creating 40000 keys takes extra time which is not necessary.

Reply:
Agree! Fewer keys are better. We have a couple of really slow tests that don't need to be so slow.